src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -239,20 +239,53 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         }
     }
 
+    private isTaggableAssetType = (asset: IAsset): boolean => {
+        return asset.type === AssetType.Image || asset.type === AssetType.VideoFrame;
+    }
+
     /**
      * Raised when the selected asset has been changed.
      * This can either be a parent or child asset
      */
     private onAssetMetadataChanged = async (assetMetadata: IAssetMetadata): Promise<void> => {
-        assetMetadata.asset.state = assetMetadata.regions.length > 0 ? AssetState.Tagged : AssetState.Visited;
+        const parentAsset = { ...(assetMetadata.asset.parent || assetMetadata.asset) };
```

Comment:
Nit: we're calling this `parentAsset` but value could be nonparent asset

Reply:
Renamed to `rootAsset` to make this more clearn.